### DISCUSSION TITLE
`repo/issues/new` url params are not supported by the app when the template isn't specified

### DISCUSSION THREAD

**Discussion by dumbasPL** [community/community#78673] [Mobile]
### Select Topic Area

Bug

### Body

related to #32208

It works fine when the repo has templates and one of the templates is specified in the URL but it doesn't when the repo has issue templates but one wasn't specified in the URL.

Steps to reproduce:
1. create a repo and add some issue templates
2. create a link like `https://github.com/<user>/<repo>/issues/new?title=foo&body=bar` and click it

On desktop, it correctly opens the new issue form with no template selected but on mobile, it just shows the template picker screen. Selecting the `No template` option doesn't help since it just opens an empty issue.

**Comment by stevepopovich** [comment]
Hi @dumbasPL thanks for writing in. I will pass this on to the team for consideration.
